the-exterminators/error-annihilator
# T230614

## Linked Issues

## PR Description

## Changes


## How to test

1. fetch Branch
2. Open, build and run the project
3. Check if project builds and looks correct on first sight
4. Test to log in using the username "admin" and the password "password"
5. Try logging out
6. Try navigating and test if all views are accesible

### Expected behaviour:

## Checklist author


## Checklist reviewer/test-engineer
